controllers/node_controller.go

```diff
@@ -123,11 +124,64 @@ func (r *NodeReconciler) doFinalize(ctx context.Context, log logr.Logger, node *
 			return ctrl.Result{}, err
 		}
 		log.Info("deleted PVC", "name", pvc.Name, "namespace", pvc.Namespace)
+
+	}
+
+	lvList := new(topolvmv1.LogicalVolumeList)
+	err = r.List(ctx, lvList, client.MatchingFields{KeyLogicalVolumeNode: node.Name})
+	if err != nil {
+		log.Error(err, "failed to get LogicalVolumes")
```

Comment:
In my opinion, it would be better to use the same error messages as in other parts.

```suggestion
		log.Error(err, "unable to fetch LogicalVolumeList")
```

Reply:
I'm sorry. In other places, it's more likely to start with `failed to ~`. You don't have to fix it.